pinc/bad_bytes.inc

```diff
@@ -898,30 +898,22 @@ function get_remarks_for_bad_byte_sequence($raw)
 
         $why_bad = "HTML numeric character reference";
     }
-    elseif(utf8_strlen($raw) == 1 && is_utf8($raw))
-    {
-        $why_bad = "Codepoint not valid for project";
-        $intended_character = "";
-    }
     else
     {
         list($code_point, $why_bad) = $_bad_byte_sequences[$raw];
     }
 
-    if(!isset($intended_character))
+    $info = $_character_data[$code_point] ?? null;
```

Comment:
What you have is fine, but you might just use a ternary:
```
$char_name = isset($_character_data[$code_point]["name"]) ? $_character_data[$code_point]["name"] : '???';
```

Reply:
Then there could be an even shorter way:

Reply:
I had to go look up the `??` operator -- very nice! Thanks for teaching me some thing new.